lib/coqProject_file.ml

```diff
@@ -225,12 +225,15 @@ let process_cmd_line ~warning_fn orig_dir parse_extra proj args =
     error "Use \"-arg -impredicative-set\" instead of \"-impredicative-set\""
 
   | "-Q" :: d :: lp :: r ->
-    let lp = if String.equal lp "Coq" then "Stdlib" else lp in
+    let lp = if String.equal lp "Coq" then "Corelib" else lp in
```

Comment:
Is `-Q Coq ...` used by anything, or is this line for consistency with the one for `-R`? In the latter case, should we drop it?

Reply:
It's here for consistency and dates back from a previous PR (i.e. it's already in master). I guess it's better if the behavior is the least surprising (even if in practice nobody ever sets `-Q Coq` so it doesn't really matter).